.github/workflows/repo-check.yml

```diff
@@ -21,7 +21,7 @@ on:
 
 jobs:
   copyright-check:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
```

Comment:
Is there a reason this is 16.04 (and in other places 18.04)? Just wondering…

Reply:
astyle...
I think I tried with 18.04 and it couldn’t find our version of astyle.

Reply:
Got it :+1:. Maybe worth adding a comment (e.g. `# needed for astyle 2.05.1`)?

Reply:
I went ahead and used 16.04 only for the style check. If it passes we are good.

Reply:
By the way, I had to do the same on Travis back when I updated to ubuntu bionic (18.04) -- it is still there (xenial = 16.04): https://github.com/mfem/mfem/blob/35a791168d798fb267ca67d1e77b6a143f28f28a/.travis.yml#L38-L45